src/client/app/graph/card/card.component.ts

```diff
@@ -109,21 +106,23 @@ export class CardComponent implements OnInit {
       const newCardPosition =
         this.canvasService.viewportCoordToCanvasCoord(this.position);
       this.card.position = {x: newCardPosition.x, y: newCardPosition.y};
-      // TODO: move all associated arrows too
-      let inArrow = this.card.incomingArrow;
-      if (inArrow) {
-        inArrow.tipPosition = {x: newCardPosition.x, y: newCardPosition.y + 40};
-        if (inArrow.fromCardId === null) {
-          inArrow.tailPosition = {x: newCardPosition.x - 50, y: newCardPosition.y + 40};
+      // Move all associated arrows too
+      this.card.incomingArrows.asArray().forEach((inArrow: Arrow) => {
+        if (inArrow) {
+          inArrow.tipPosition = {x: newCardPosition.x, y: newCardPosition.y + this.card.size.height/2};
```

Comment:
Can we change all of the object literals (throughout) to be indented consistently?